accounts-tequila-client.js

```diff
@@ -28,19 +28,19 @@ function __(/* args */) {
 // Documented in accounts-tequila-server.js
 function start (opts) {
   if (! opts) opts = {}
-  var queryString = window.location.search
+
+  const queryString = new URLSearchParams(window.location.search);
```

Comment:
Please defend the good-enough availability of `URLSearchParams` in a comment (ie-trollface.gif)